lib/server-process.js

```diff
@@ -34,8 +19,7 @@ const createSbot = require('secret-stack')()
   .use(require('ssb-about'))
   .use(require('ssb-private'))
   .use(require('ssb-room/tunnel/client'))
-  // .use(require('ssb-dht-invite')) // this one must come before dhtTransport
-  // .use(dhtTransport)
+  .use(require('ssb-dht-invite'))
```

Comment:
I always wondered why the chaining order is like it is here as it is not arbitrary, right? It would be really great to add some documentation about that here at some point.

Reply:
It's not arbitrary, but it's also not documented. To be candid, I think the SecretStack plugin system is an anti-pattern that encourages global mutable state. I'm doing my best to randomize the plugin order in Oasis with this pattern (https://github.com/fraction/flotilla/blob/master/index.js) but it's still a mess. I really wish these modules would just `require()` their dependencies instead of using another dependency system.

Reply:
> To be candid, I think the SecretStack plugin system is an anti-pattern that encourages global mutable state.

:scream: hot take! I actually like the secret-stack plugin system, out of all the other ideas in the SSB stack.

Reply:
If y'all want some insight on what depends on what, I wrote it down for Manyverse here: https://github.com/staltz/manyverse/blob/3a4e80d151fe70cbd7d9fcfe133b1eec863aef14/src/backend/ssb.ts#L57-L94